kinova_tracking/IK_Kinova.py

```diff
@@ -41,17 +47,19 @@ def objective_function(x, biorbd_model, q_ik_thorax, table_markers, thorax_marke
 
         out4 = 0
         for h in range(1, 3):
-            out4 += (x[-h] - 0.) ** 2
+            out4 += (x[-h] - 0.0) ** 2
 
         return 1000 * table5_xyz + 1000 * table6_xy + out2 + mark_out + 10 * out3 + out4
 
     q = np.zeros((biorbd_model.nbQ(), markers.shape[2]))
-    bounds = [(mini, maxi) for mini, maxi in zip(utils.get_range_q(biorbd_model)[0], utils.get_range_q(biorbd_model)[1])]
+    bounds = [
+        (mini, maxi) for mini, maxi in zip(utils.get_range_q(biorbd_model)[0], utils.get_range_q(biorbd_model)[1])
+    ]
     for f in range(markers.shape[2]):
         x0 = q0 if f == 0 else q[:, f - 1]
         pos = optimize.minimize(
             fun=objective_function,
-            args=(biorbd_model, q_ik_1[: , f], markers[:, 14:, f], markers[:, 0:14, f]),
+            args=(biorbd_model, q_ik_1[:, f], markers[:, 14:, f], markers[:, 0:14, f]),
```

Comment:
hard coded ?

can you write the index before the loop ?
idx_markers_human_body = ...
idx_revolute_joint = ...
args=(biorbd_model, q_ik_1[:, f], markers[:, idx_revolute_joint, f], markers[:, idx_markers_human_body, f]),

Reply:
done